ts/boot/configureStoreAndPersistor.ts

```diff
@@ -422,7 +422,11 @@ const migrations: MigrationManifest = {
   // Version 32
   // Removes new Messages Home FF
   "32": (state: PersistedState) =>
-    omit(state, "persistedPreferences.isNewHomeSectionEnabled")
+    omit(state, "persistedPreferences.isNewHomeSectionEnabled"),
+  // Version 33
+  // Removes it wallet section FF
+  "33": (state: PersistedState) =>
+    omit(state, "persistedPreferences.isItWalletTestEnabled")
 };
```

Comment:
For this to work we have to raise the `CURRENT_REDUX_STORE_VERSION` constant.

Reply:
Good catch! Thank you!